hack/cleanup.sh

```diff
@@ -32,11 +32,15 @@ sudo bash -c '
     echo "Killing conmon, pause processes"
     pkill -9 conmon
     pkill -9 pause
-
-
```

Comment:
may be we should start using the crio wipe here?

Reply:
fixed.